templates/zerver/help/invite-new-users.md

```diff
@@ -1,162 +1,90 @@
 # Invite new users
 
-There are a number of ways to grant access to your Zulip organization;
-the article below describes each in more detail.
+When you invite users, you can:
 
-* Allow **anyone to join** without an invitation.
+* Set the [role](/help/roles-and-permissions) that they will have when
+  they join.
 
-* Allow people to join based on the **domain** of their email address.
+* Configure the streams they will automatically be added to.
 
-* Send **email invitations**.
+Your organization may also want to configure default
+[streams](/help/set-default-streams-for-new-users) and
+[settings](/help/configure-default-new-user-settings) for new users.
 
-* Share a **reusable invitation link**.
 
-The last two, invite-based, techniques also allow you to control the
-[role (owner, admin, moderator, member, or guest)](/help/roles-and-permissions) that the
-invited people will have.
+## Send e-mail invitations
 
-You can also manage access by
-[controlling how users authenticate](/help/configure-authentication-methods)
-to Zulip.  For example, you could allow anyone to join without an
-invitation, but require them to authenticate via LDAP.
-
-## Enable email signup
+!!! warn ""
+    You will only see an **Invite users** option if you have permission to
+    invite users to the organization.
```

Comment:
Should "have permission" be a link?

Reply:
Sure, it can link to `/help/restrict-account-creation#change-who-can-send-invitations`.

Reply:
Added such a link.